tests/test_synced_collection.py

```diff
@@ -0,0 +1,587 @@
+# Copyright (c) 2020 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+import pytest
+import uuid
+import os
+import json
+from tempfile import TemporaryDirectory
+from collections.abc import MutableMapping
+from collections.abc import MutableSequence
+
+from signac.core.synced_list import SyncedCollection
+from signac.core.jsoncollection import JSONDict
+from signac.core.jsoncollection import JSONList
+from signac.errors import InvalidKeyError
+from signac.errors import KeyTypeError
+
+try:
+    import numpy
+    NUMPY = True
+except ImportError:
+    NUMPY = False
+
+FN_JSON = 'test.json'
+
+
+@pytest.fixture
+def testdata():
+    return str(uuid.uuid4())
+
+
+class TestSyncedCollectionBase():
+
+    @pytest.fixture(autouse=True)
+    def synced_collection(self):
+        self._tmp_dir = TemporaryDirectory(prefix='jsondict_')
+        self._fn_ = os.path.join(self._tmp_dir.name, FN_JSON)
+        yield
+        self._tmp_dir.cleanup()
+
+    def test_from_base(self):
+        print(JSONDict.backend)
```

Comment:
Sorry for the mistake. I'll make sure to check every single line of code before creating the commit. This commit actually changed the whole file, so I must have missed it. It will not happen again.